requirements.txt

```diff
@@ -26,7 +26,6 @@ pycrypto==2.6.1
 py-sr25519-bindings==0.1.2
 py-ed25519-bindings==0.1.1
 py-bip39-bindings>=0.1.6
-pytest-asyncio
```

Comment:
How come we took out pytest-asyncio? Any reason?

Reply:
Oh, that shouldn't be there, I'll put it back